.github/labeler.yml

```diff
@@ -10,13 +10,6 @@ area/automation:
   - src/Uno.UI.Wasm.Tests/*
   - src/Uno.UI.Wasm.Tests/**/*
 
-area/android:
-  - src/Uno.UI.BindingHelper.Android/*
-  - src/Uno.UI.BindingHelper.Android/**/*
-  - '**/*.Android.cs'
-  - '**/*.android.cs'
-  - '**/*.droid.cs'
```

Comment:
The correct label is `platform/android`, not `area/android` (same goes for other platforms).

I moved those incorrect labels down (to keep `area/*` labels contiguous). I also removed the `*.droid.cs` suffix which isn't used. Also removed `.android.cs` and opened #10227 to rename files to have consistent suffix.

Reply:
Makes sense. Thanks!